oneLiners.py

```diff
@@ -0,0 +1,130 @@
+import functools
+
+
+############################################## 1.1 ##############################################
+
+
+def double_letter(my_str) -> str:
```

Comment:
```suggestion
def double_letter(my_str: str) -> str:
```
Missing type hints for all your parameters